File: draw_figures/plotMarker.m
function plotMarker(x, y)

plot([x-0.08 x+0.08], [y y], 'k-', 'LineWidth',1.5);
plot(x, y, '^', 'Color','k', 'MarkerFaceColor', 'k', 'MarkerSize',8);

end

File: Python_Ver/draw_figures/plotMarker.py
import matplotlib.pyplot as plt

def plotMarker(x, y):
    """
    Plot a marker with a horizontal line.

    Parameters:
    x : float
        X-coordinate of the marker.
    y : float
        Y-coordinate of the marker.
    """
    plt.plot([x - 0.08, x + 0.08], [y, y], 'k-', linewidth=1.5)  # Horizontal line
    plt.plot(x, y, '^', color='k', markerfacecolor='k', markersize=8)  # Triangle marker
